src/query/ts/exemplar.go

```diff
@@ -0,0 +1,10 @@
+package ts
```

Comment:
Unfortunately will need the `// Copyright (c) 2019 Uber Technologies, Inc.` header